### DISCUSSION TITLE
New Blueprint: Cycle Automations

### DISCUSSION THREAD

**Discussion by aderusha** [HASwitchPlate/HASPone#142] [Show and tell]
[![Cycle Automations](https://my.home-assistant.io/badges/blueprint_import.svg)](https://my.home-assistant.io/redirect/blueprint_import/?blueprint_url=https%3A%2F%2Fgithub.com%2FHASwitchPlate%2FHASPone%2Fblob%2Fmain%2FHome_Assistant%2Fblueprints%2Fhasp_Cycle_Automations.yaml)

![Preview](https://raw.githubusercontent.com/HASwitchPlate/HASPone/main/images/hasp_Cycle_Automations.gif)

A button on the HASP will toggle through as many as 10 selected automations.  This allows the user to assign multiple blueprints to the same button on the HASPone device, and to cycle between them by pressing the selected button.

Optionally, a timeout can be set to cycle back to a "default" automation after a specified interval, or to continuously cycle through selected automations.

**Comment by IIAIronWolf** [comment]
Sweet! This will be great for getting further detail on, say, a weather button.
